smart-contracts/contracts/BridgeProxy/deposit/ArbitrumOrbitNativeDepositBridgeProxy.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+import {BridgeProxy} from "../BridgeProxy.sol";
+import {IL1GatewayRouter} from "../../interfaces/arb/IArbL1GatewayRouter.sol";
+import {IInbox} from "../../interfaces/arb/IInbox.sol";
+
+contract ArbitrumOrbitNativeDepositBridgeProxy is BridgeProxy {
+  error AssetMismatch(address expected, address actual);
+
+  IInbox public immutable INBOX =
+    IInbox(0x0000000000000000000000000000000000000064);
+  IL1GatewayRouter public immutable ROUTER;
+
+  constructor(
+    address l1GatewayRouter,
+    uint256 relayPoolChainId,
+    address relayPool,
+    address l2BridgeProxy
+  ) BridgeProxy(relayPoolChainId, relayPool, l2BridgeProxy) {
+    ROUTER = IL1GatewayRouter(l1GatewayRouter);
+  }
+
+  function bridge(
+    address asset, //l2 token
+    address l1Currency, //l1 token
+    uint256 amount,
+    bytes calldata /* txParams */,
+    bytes calldata /* extraData */
+  ) external payable override {
+    if (l1Currency == address(0)) {
+      // simple deposit wont work as it deposits to an alias by default
+      // we have to create a retryable ticket to deposit to the L1 bridge proxy
+      INBOX.createRetryableTicket(
+        L1_BRIDGE_PROXY, // to
```

Comment:
huh, can we replce with `VAULT_CHAIN_BRIDGE_PROXY` or does it mean we have to replace everywhere? If so please add comments to clarify otherwise we will get very confused!